dev-support/mini-submarine/README.md

```diff
@@ -52,6 +52,26 @@ mvn clean install package -DskipTests
 cd submarine/dev-support/mini-submarine/
 ./build_mini-submarine.sh
 ```
+#### Package An Existing Release Candidates
+When doing release, the release manager might needs to package a artifact candidatesin this docker image and public the image candidate for a vote.
```

Comment:
`candidatesin` should be `candidates in`

Reply:
Thanks for catching this.